js/menu.js

```diff
@@ -223,8 +224,21 @@ const displayPopup = () => {
   });
 };
 
+const validation = () => {
+  const span = document.createElement('span');
+  span.textContent = 'invalid emai';
+  span.classList.add('text-red');
+  span.classList.add('m-l-5');
+  email.addEventListener('input', () => {
+    if (email.validity.typeMismatch) {
```

Comment:
* If the validation is not OK, you show an error message to the user near the submit button informing them of the error and the form is not sent.

Reply:
that's what I did, can you check well please

Reply:
* Kindly make the error message appear near the submit  button
![image](https://user-images.githubusercontent.com/52497246/132569387-71b4672f-3c6d-47d3-9fea-3f79ee56b1c4.png)